plugin/gv.vim

```diff
@@ -297,14 +297,19 @@ function! s:gl(buf, visual)
   call setloclist(0, insert(getloclist(0), {'bufnr': a:buf}, 0))
```

Comment:
Why not set the title here with the other call to setloclist?

`:help setqflist` says: 
> Returns zero for success, -1 for failure.

Can this actually throw an error?

Reply:
Good point.  It should try to set it there already probably, but handle the error(s) for when it is not supported.